package.json

```diff
@@ -56,13 +57,14 @@
     "pg-hstore": "^2.3.4",
     "sequelize": "^6.17.0",
     "sequelize-cli": "^6.4.1",
+    "supertest": "^6.2.2",
     "swagger-jsdoc": "^6.1.0",
-    "swagger-ui-express": "^4.3.0"
+    "swagger-ui-express": "^4.3.0",
+    "uuid": "^8.3.2"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.17.0",
     "@babel/register": "^7.8.3",
-    "eslint": "8.1.0",
+    "eslint": "7.32.0",
```

Comment:
is this change the one introducing "babel-eslint not found" from hound-ci

Reply:
Tried both in a previous task, changed back when rebasing. Further digging led to show Hound doesn't support that parser yet